sql/search_derived/mobile_search_clients_daily_v1/fenix_metrics.template.sql

```diff
@@ -1,14 +1,14 @@
 -- baseline for {namespace} ({app_name} {channel})
-baseline_{namespace} AS (
+baseline_ { namespace } AS (
```

Comment:
You may want to ignore all of the templated files, since the formatting script doesn't handle brackets in this context well.

Reply:
Good catch! I added the template files to the `SKIP` list